Logger.Tests/BaseLoggerMixinsTests.cs

```diff
@@ -8,14 +8,11 @@ namespace Logger.Tests;
 public class BaseLoggerMixinsTests
 {
     [TestMethod]
-    [ExpectedException(typeof(ArgumentNullException))]
     public void Error_WithNullLogger_ThrowsException()
     {
         // Arrange
-
         // Act
-        //BaseLoggerMixins.Error(null, "");
-
+        Assert.ThrowsExactly<ArgumentNullException>(() => BaseLoggerMixins.Error(null!, ""));
         // Assert
```

Comment:
nit: structure could be a little neater. This is what I do, but the extra whitespace is my own preference:
```suggestion
// Arrange

// Act

// Assert
Assert.ThrowsExactly<ArgumentNullException>(() => BaseLoggerMixins.Error(null!, ""));
```